src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeCompletion.cs

```diff
@@ -88,7 +88,7 @@ public void Reset()
             _state = null;
```

Comment:
this assignment to `_state` means that readonly can't be used here.

Reply:
![image](https://user-images.githubusercontent.com/35951936/116047756-1944e380-a67d-11eb-91fa-b08ec83e3c17.png)
There is no ```readonly``` at ```Reset()```. Github diff merged ```IsCompletedOrThrow()``` and ```Reset()``` :)

Reply:
Ah, my bad, unhelpful diff window size.